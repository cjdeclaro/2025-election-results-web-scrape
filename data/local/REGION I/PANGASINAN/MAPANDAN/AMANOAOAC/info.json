{"regions":[{"categoryCode":null,"masterCode":null,"code":"55280004","name":"55280004"},{"categoryCode":null,"masterCode":null,"code":"55280005","name":"55280005"}]}