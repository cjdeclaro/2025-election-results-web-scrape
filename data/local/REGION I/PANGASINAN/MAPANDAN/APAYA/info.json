{"regions":[{"categoryCode":null,"masterCode":null,"code":"55280006","name":"55280006"},{"categoryCode":null,"masterCode":null,"code":"55280007","name":"55280007"}]}