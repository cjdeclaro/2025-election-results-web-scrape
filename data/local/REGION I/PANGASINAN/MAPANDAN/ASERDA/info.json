{"regions":[{"categoryCode":null,"masterCode":null,"code":"55280008","name":"55280008"},{"categoryCode":null,"masterCode":null,"code":"55280009","name":"55280009"}]}