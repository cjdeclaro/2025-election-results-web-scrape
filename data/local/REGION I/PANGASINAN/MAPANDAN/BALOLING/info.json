{"regions":[{"categoryCode":null,"masterCode":null,"code":"55280010","name":"55280010"},{"categoryCode":null,"masterCode":null,"code":"55280011","name":"55280011"},{"categoryCode":null,"masterCode":null,"code":"55280012","name":"55280012"},{"categoryCode":null,"masterCode":null,"code":"55280013","name":"55280013"}]}