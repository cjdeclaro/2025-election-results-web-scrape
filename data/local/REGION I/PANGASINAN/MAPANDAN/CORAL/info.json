{"regions":[{"categoryCode":null,"masterCode":null,"code":"55280014","name":"55280014"},{"categoryCode":null,"masterCode":null,"code":"55280015","name":"55280015"}]}