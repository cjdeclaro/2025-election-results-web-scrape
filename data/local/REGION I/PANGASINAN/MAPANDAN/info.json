{"regions":[{"categoryCode":"5","masterCode":"5528000","code":"5528001","name":"AMANOAOAC"},{"categoryCode":"5","masterCode":"5528000","code":"5528002","name":"APAYA"},{"categoryCode":"5","masterCode":"5528000","code":"5528003","name":"ASERDA"},{"categoryCode":"5","masterCode":"5528000","code":"5528004","name":"BALOLING"},{"categoryCode":"5","masterCode":"5528000","code":"5528006","name":"CORAL"},{"categoryCode":"5","masterCode":"5528000","code":"5528007","name":"GOLDEN"},{"categoryCode":"5","masterCode":"5528000","code":"5528008","name":"JIMENEZ"},{"categoryCode":"5","masterCode":"5528000","code":"5528009","name":"LAMBAYAN"},{"categoryCode":"5","masterCode":"5528000","code":"5528010","name":"LUYAN"},{"categoryCode":"5","masterCode":"5528000","code":"5528012","name":"NILOMBOT"},{"categoryCode":"5","masterCode":"5528000","code":"5528013","name":"PIAS"},{"categoryCode":"5","masterCode":"5528000","code":"5528014","name":"POBLACION"},{"categoryCode":"5","masterCode":"5528000","code":"5528015","name":"PRIMICIAS"},{"categoryCode":"5","masterCode":"5528000","code":"5528016","name":"SANTA MARIA"},{"categoryCode":"5","masterCode":"5528000","code":"5528018","name":"TORRES"}]}