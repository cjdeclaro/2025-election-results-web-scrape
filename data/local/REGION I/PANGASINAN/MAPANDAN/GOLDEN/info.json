{"regions":[{"categoryCode":null,"masterCode":null,"code":"55280016","name":"55280016"},{"categoryCode":null,"masterCode":null,"code":"55280017","name":"55280017"}]}